packages/stk/stk_unit_tests/stk_balance/UnitTestLoadBalanceMultipleCriteria.cpp

```diff
@@ -60,9 +60,9 @@ class MultipleCriteria : public stk::unit_test_util::MeshFixture
             stk::mesh::Field<double> &criteria2 = get_meta().declare_field<stk::mesh::Field<double>>(stk::topology::ELEM_RANK, "criteria2");
             stk::mesh::Field<double> &procId = get_meta().declare_field<stk::mesh::Field<double>>(stk::topology::ELEM_RANK, "proc_id");
 
-            stk::mesh::put_field_on_mesh(criteria1, get_meta().universal_part(), nullptr);
-            stk::mesh::put_field_on_mesh(criteria2, get_meta().universal_part(), nullptr);
-            stk::mesh::put_field_on_mesh(procId, get_meta().universal_part(), nullptr);
+            stk::mesh::put_field_on_mesh(criteria1, get_meta().universal_part(), (double*)nullptr);
```

Comment:
See comment above on C-style casts.

Reply:
I agree Mark, static_cast is better. I've pushed a new commit to switch these to static_cast.